src/EnergyPlus/HVACUnitarySystem.cc

```diff
@@ -3023,38 +3052,39 @@ namespace HVACUnitarySystem {
 					ErrorsFound = true;
 				}
 			}
+			DesignSpecMSHP( DesignSpecNum ).NoLoadAirFlowRateRatio = Numbers( 3 );
 
 			// initialize number of speeds (MIN = 4 or greater of cooling and heating speeds x 2)
 			MaxSpeedNum = 0; // set to 0 so we know if the inputs are valid
 
-			if ( mod( double( NumNumbers - 2 ), 2.0 ) > 0.0 ) {
+			if ( mod( double( NumNumbers - 3 ), 2.0 ) > 0.0 ) {
```

Comment:
Interesting.  I know you didn't add it here, so no big deal, but I'm not used to seeing `mod` as a function. Usually an operator is used:

```
(NumNumbers-3) % 2 > 0
```

Nothing to worry about here, just a new one for me.